azkaban/__main__.py

```diff
@@ -421,6 +427,19 @@ def schedule_workflow(project_name, _date, _time, _span, _flow, _job, _url,
     'Flow %s scheduled successfully.\n' % (_flow, )
   )
 
+def sla(_url, _alias, _schedule, _delay, _rule, _email, _kill):
+  """Set schedule's SLA"""
+  session = _get_session(_url, _alias)
+  kwargs = {
```

Comment:
Let's avoid creating this dictionary explicitly by calling the method with named parameters directly:

```python
session.sla(
  schedule=_schedule,
  delay=_delay,
  # ...
)
```

(The method above doesn't do this because the set of passed arguments changes.)